jdi-light-mobile-examples/src/test/java/io/github/epam/tests/recommended/TableTests.java

```diff
@@ -0,0 +1,116 @@
+    }
+    
+    private void tablePerformance(Table table) {
+        assertEquals(table.size(), 4);
+        assertEquals(table.count(), 400);
+        assertEquals(table.header(), asList("Name", "Phone", "Email", "City"));
+        start();
+        assertEquals(table.row(1).getValue(),
+                "Burke Tucker;076 1971 1687;et.euismod.et@ut.edu;GozŽe");
+        logTime("Get 1 row");
+        // TODO fix performance
+        assertEquals(table.row("Burke Tucker").getValue(),
+                "Burke Tucker;076 1971 1687;et.euismod.et@ut.edu;GozŽe");
+        logTime("Get 'Burke Tucker' row");
+
+        String zacharyEmail = "ipsum.non.arcu@auctorullamcorper.ca";
+        assertEquals(table.cell(3,4), zacharyEmail);
+        logTime("Get cell(3,4)");
+        assertEquals(table.cell("Email",4), zacharyEmail);
+        logTime("Get cell(Email,4)");
+        assertEquals(table.cell(3,"Zachary Hendrix"), zacharyEmail);
+        // TODO fix performance
+        logTime("Get cell(3,Zachary Hendrix)");
+        assertEquals(table.cell("Email","Zachary Hendrix"), zacharyEmail);
+        // TODO fix performance
+        logTime("Get cell(Email,Zachary Hendrix)");
+
+        // TODO fix performance
+        assertEquals(table.column(2).getValue().substring(0, 30),
+                "076 1971 1687;(011307) 16843;0");
+        // TODO fix performance
+        logTime("Get column(2)");
+
+        // TODO fix performance
+        assertEquals(table.column("Phone").getValue().substring(0, 30),
+                "076 1971 1687;(011307) 16843;0");
+        // TODO fix performance
+        logTime("Get column(Phone)");
+
+        String value = table.preview();
+        assertEquals(value.substring(0,194),
+        "Name Phone Email City" +
+            "Burke Tucker 076 1971 1687 et.euismod.et@ut.edu GozŽe" +
+            "Grady Brock (011307) 16843 cursus.et@commodo.org Alcobendas" +
+            "Harding Lloyd 0800 1111 neque.In.ornare@mauris.co.uk Beauvais");
+        logTime("Preview");
+        /* Takes too much time for tests, but you can run this code by yourself
+        value = table.getValue();
+        assertEquals(value.substring(0,228),
+            "||X||Name|Phone|Email|City||" + LINE_BREAK +
+            "||1||Burke Tucker|076 1971 1687|et.euismod.et@ut.edu|GozŽe||" + LINE_BREAK +
+            "||2||Grady Brock|(011307) 16843|cursus.et@commodo.org|Alcobendas||" + LINE_BREAK +
+            "||3||Harding Lloyd|0800 1111|neque.In.ornare@mauris.co.uk|Beauvais||");
+        logTime("Get value");*/
+    }
+
+    @Test
+    public void tableDataTest() {
+        assertEquals(users.row(2).asData(UserInfo.class),
+                GRADY_BROCK);
+    }
+
+    @Test
+    public void tableEntityTest() {
+        UserRow user = users.row(2).asLine(UserRow.class);
+        user.name.click();
+        validateAlert(containsString("Brock"));
+        user.city.click();
+        validateAlert(is("Alcobendas"));
+    }
+
+    private static long timeStart;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5104364)